walk-in-interview/src/main/webapp/show-job-posts-made.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
```

Comment:
I think the path should be /webapp/business-jobs-list/index.html (similarly for the script file)